### PR TITLE
Add xsv

### PR DESCRIPTION
**Pull request by rvalieris**
migrating recipe from bioconda.

ping @bgruening, @johanneskoester , added as maintainers.

### CODE REVIEW

File: recipes/xsv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  name: xsv
+  version: {{ version }}
+
+source:
+  url: https://github.com/BurntSushi/xsv/archive/{{ version }}.tar.gz
+  sha256: 2b75309b764c9f2f3fdc1dd31eeea5a74498f7da21ae757b3ffd6fd537ec5345
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (synapticarbors) [inline recipes/xsv/meta.yaml]: Any explicit reason to skip windows? I see that the source repo is running on appveyor.

File: recipes/xsv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  host:
+    - rust
+
+test:
+  commands:
+    - xsv --help > /dev/null
+
+about:
+  home: https://github.com/BurntSushi/xsv
+  license: MIT
```
Comment (synapticarbors) [inline recipes/xsv/meta.yaml]: If you're going to distribute this under the `MIT` license, please include the `license_file`, which should be in the source distribution.

File: recipes/xsv/meta.yaml (hunk 1 above)
Comment (rvalieris) [reply recipes/xsv/meta.yaml]: for some reason I though rust was not available on windows, I see it is, I will give it a try.

File: recipes/xsv/meta.yaml (hunk 2 above)
Comment (rvalieris) [reply recipes/xsv/meta.yaml]: alright I will add it.

**Comment by rvalieris** [comment]
build passed !

File: recipes/xsv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  license_file: LICENSE-MIT
+  summary: A fast CSV toolkit written in Rust.
+
+extra:
+  recipe-maintainers:
+    - bgruening
+    - rvalieris
+    - johanneskoester
```
Comment (marcelotrevisani) [inline recipes/xsv/meta.yaml]: Does everyone agree to be a maintainer?

@bgruening , @johanneskoester

Comment (rvalieris) [reply recipes/xsv/meta.yaml]: it should be fine, I can remove them later if they want right ?

Comment (bgruening) [reply recipes/xsv/meta.yaml]: Ok with me. Johannes does not follow conda-forge but is generally ok I think :)

**Pull request merged**